src/main/features/response/tasks/decideHowYouWillPayTask.ts

```diff
@@ -25,6 +25,8 @@ export class WhenWillYouPayTask {
         && this.explanationIsValidIfRequired(responseDraft.payBySetDate)
     } else if (responseDraft.defendantPaymentOption.option === DefendantPaymentType.INSTALMENTS) {
       return isValid(responseDraft.defendantPaymentPlan)
+    } else if (responseDraft.defendantPaymentOption.option === DefendantPaymentType.IMMEDIATELY) {
+      return isValid(responseDraft.defendantPaymentOption)
```

Comment:
This will run validation second time. Please use switch statement and break early for `DefendantPaymentType.IMMEDIATELY` option.